src/httpInterceptor.js

```diff
@@ -7,12 +7,15 @@ function httpEtagInterceptorFactory () {
     if (itemCache) {
       var cacheInfo = itemCache.info()
       var cacheResponseData = cacheInfo.cacheResponseData
+      var cacheResponseHeaders = cacheInfo.cacheResponseHeaders
       var etag = response.headers().etag
       var cacheData = {}
 
       if (etag) {
         cacheData.etagHeader = etag
         if (cacheResponseData) cacheData.responseData = response.data
+        // TODO: Allow user to speficy which headers to cache?
+        if (cacheResponseHeaders) cacheData.responseHeaders = response.headers
```

Comment:
Hmm, I'll have to look deeper into it then. I'll try to cache the values returned from the function call, and return them as a function to the `.cached` method.


I'll try to get tests in place this weekend.